source/objects/RenderTarget.cpp

```diff
@@ -209,6 +217,174 @@ Pixel RenderTarget::SampleBilinear(float u, float v, uint8_t mip) const
 	};
 }
 
+bool RenderTarget::Load(const char* filepath, bool generateMips, bool scaleToRT)
+{
+	if (!IsValid()) return false;
+
+	int imageWidth = 0;
+	int imageHeight = 0;
+	int channels = CHANNELS[static_cast<uint8_t>(GetFormat())];
+	size_t stride = STRIDES[static_cast<uint8_t>(GetFormat())];
+	
+	uint8_t* rtData = GetRawData();
+	const bool isFloat = stride == sizeof(float);
+
+	if (isFloat) {
+		float* imageData = stbi_loadf(filepath, &imageWidth, &imageHeight, nullptr, channels);
```

Comment:
You're not even localising the filepath to data/vistrace, never mind the fact you're not sandboxing it